src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ReflectionXmlFormatWriter.cs

```diff
@@ -244,54 +273,60 @@ private void ReflectionWriteValue(Type type, object value, bool writeXsiType)
                 }
                 else
                 {
-                    throw new NotImplementedException();
+                    ReflectionInternalSerialize(value, value.GetType().TypeHandle.Equals(memberType.TypeHandle), writeXsiType, memberType);
                 }
             }
             else
             {
                 if (isNullableOfT)
                 {
-                    throw new NotImplementedException();
+                    if (memberValue == null)
+                    {
+                        Type[] genericArgumentTypes = memberType.GetGenericArguments();
+                        if (genericArgumentTypes.Length != 1)
+                        {
+                            throw new InvalidOperationException(string.Format("Cannot serialize type: {0}", memberType.Name));
```

Comment:
Not blocking, but for future I recommend we consider (1) localizing the string, and (2) having an error message that says why this member type cannot be serialized.  How will the user know what is wrong? Here it seems to be saying it cannot serialize nullable types unless they have only one generic parameter.